### ISSUE TITLE
strand goals - usar tab para selecionar o elemento curricular sugerido

### ISSUE THREAD

**Issue by gabrielabonicio** [camino-school/lanttern#83]

**Comment by endoooo** [comment]
@gabrielabonicio for some interactions, we have guidelines on how keyboard interaction should behave, which is important for accessibility (a11y). the curriculum search is a combobox — although the implementation is not 100%, this is what we are trying to achieve: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Combobox_Role#keyboard_interactions

the Tab key should not be linked to the item selection, because it's used to move between form fields.
but you can use the Enter key, which should work.
